src/lib/y2storage/proposal/devicegraph_generator.rb

```diff
@@ -87,6 +88,17 @@ def devicegraph(volumes, initial_graph, disk_analyzer)
 
     protected
 
+      # Modifies planned volumes adding an encryption password.
+      # Only volumes that can_live_on_logical_volume could have
+      # an encryption password.
+      # @param volumes [PlannedVolumesList]
+      def add_encryption_password(volumes)
+        return volumes unless settings.use_encryption
+        volumes.select(&:can_live_on_logical_volume).map do |volume|
```

Comment:
Strictly speaking, I guess so. But that would probably mean three contexts with the same expectations. A little bit too much. :-)  I guess we can skip it in order to get shorter output. If LVM and encrypted are both equal, I guess everybody can guess that LVM+encrypt is also de same.